packages/orchestrator-ui-components/src/rtk/endpoints/startOptions.ts

```diff
@@ -53,7 +53,9 @@ export type StartOptionsResponse<StartOption> = {
     startOptions: StartOption[];
 };
 
-type TaskOption = Pick<WorkflowDefinition, 'name' | 'description'>;
+type TaskOption = Pick<WorkflowDefinition, 'name'> & {
```

Comment:
Isn't making the description non optional here a bit hacky? If we say a description may or may not be part of a workflow definition than it's hard to assume it will alway be present here right? Did you try making it non-optional on the WorkflowDefintion type to see what happens?

Reply:
Agree, `discuss`